RELEASE.md

```diff
@@ -0,0 +1,49 @@
+# Releasing
```

Comment:
generally makes sence, so can split out quality gates as reference to other doc, because depending on what is released it may or may have various gates 

https://github.com/ComposableFi/composable/blob/main/rfcs/0006-picasso-runtime-upgrades.md

for me release is git commit accepted as releasable and relevant wasm +reference from it + diff (with accompanied release notes). + binary node built from commit.  likely node only needs semantic versioning, not sure about wasm. also there is sdk. but imagine we have repo builds, then git commit is root of all, semantic versions are just comments (release notes = notes + version comment)

Reply:
For just the parachain related stuff, that is correct. Note that the actual version of the runtime is determined by the `impl_version` and the `spec_version`. Ideally we stop hardcoding those, and generate them during build time using the semantic version.